src/signalrclient/hub_connection_impl.cpp

```diff
@@ -38,6 +38,22 @@ namespace signalr
 
     pplx::task<void> hub_connection_impl::start()
     {
+        if (m_proxies.size() > 0)

```

Comment:
Maybe it would be good to log as info that the client will not be subscribed to any hubs if there are no proxies.